### PR TITLE
Create /getgamemode

### LINKED ISSUE #1839: get a players gamemode
Simple. /getgamemode. Get's the player's gamemode so that administrative action may be used :D 
I already made the command and tested it, just wanted thoughts (and you have to post an issue before making a pr.)

### PR DESCRIPTION
**Pull request by notanoperator**
Fixes #1839 
Get's players gamemode.

### CODE REVIEW

File: src/main/java/me/totalfreedom/totalfreedommod/command/Command_getgamemode.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+@CommandPermissions(level = Rank.SUPER_ADMIN, source = SourceType.BOTH)
+@CommandParameters(description = "Get a players gamemode", usage = "/<command> <player>", aliases = "ggm,getgm,isgmc")
+public class Command_getgamemode extends FreedomCommand
+{
+
+    @Override
+    public boolean run(CommandSender sender, Player playerSender, Command cmd, String commandLabel, String[] args, boolean senderIsConsole)
+    {
+       if (args.length < 1) {
```
Comment (lemonsked) [inline src/main/java/me/totalfreedom/totalfreedommod/command/Command_getgamemode.java:17]: Formatting.

File: src/main/java/me/totalfreedom/totalfreedommod/command/Command_getgamemode.java [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+           return false;
+       }
+       
+       final Player player = getPlayer(args[0]);
+       
+       if (player == null) {
```
Comment (lemonsked) [inline src/main/java/me/totalfreedom/totalfreedommod/command/Command_getgamemode.java:23]: Formatting.

File: src/main/java/me/totalfreedom/totalfreedommod/command/Command_getgamemode.java [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+           msg(ChatColor.GRAY + "Player not found!");
+       }
+       
+       msg(ChatColor.RED + player.getName() + "'s gamemode is " + ChatColor.AQUA + player.getGameMode() + ChatColor.RED + "!");
+    return true;
```
Comment (lemonsked) [inline src/main/java/me/totalfreedom/totalfreedommod/command/Command_getgamemode.java:28]: Formatting.

File: src/main/java/me/totalfreedom/totalfreedommod/command/Command_getgamemode.java [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+       final Player player = getPlayer(args[0]);
+       
+       if (player == null) {
+           msg(ChatColor.GRAY + "Player not found!");
+       }
+       
+       msg(ChatColor.RED + player.getName() + "'s gamemode is " + ChatColor.AQUA + player.getGameMode() + ChatColor.RED + "!");
+    return true;       
+    }
+
```
Comment (lemonsked) [inline src/main/java/me/totalfreedom/totalfreedommod/command/Command_getgamemode.java:30]: Useless extra line.

**Comment by TheWadeSmith** [comment]
Just do /who (player)

**Comment by notanoperator** [comment]
"Needs Testing" "Needs Changes"
Codes fine.
Works fine.

**Comment by mayokek** [comment]
You'll need to fix the formatting as said above.

**Comment by Wild1145** [comment]
As the issue has not yet been accepted or assigned a milestone this PR is also on hold until further notice.

**Comment by ghost** [comment]
Just use /who (player)...

**Review by Wild1145 — CHANGES_REQUESTED** [review]
Please address the formatting issues raised within the inline comments.

**Comment by Wild1145** [comment]
Thanks for contributing. I am closing this issue for inactivity. Should you have anything further to add in the future please either re-open the issue yourself or tag a member of staff and we will re-open the issue for further discussion.

**Pull request closed without merging**